tests/workflows/test_imbalanced_join.py

```diff
@@ -0,0 +1,23 @@
+"""
+This data represents a skewed but realistic dataset that dask has been struggling with in the past.
+Workflow based on https://github.com/coiled/imbalanced-join/blob/main/test_big_join_synthetic.ipynb
+"""
+import dask.dataframe as dd
+import pytest
+
+
+@pytest.mark.client("imbalanced_join")
+def test_merge(client):
+    """Merge large df and small df"""
+    large_df = dd.read_parquet("s3://test-imbalanced-join/df1/key_000*.parquet")
```

Comment:
This is temporarily limiting how much data is being read. TODO: remove limiting.